frontend/packages/kubevirt-plugin/src/components/vms/vm-resource.tsx

```diff
@@ -0,0 +1,98 @@
+import * as React from 'react';
+
+import {
+  getDescription,
+  getOperatingSystemName,
+  getOperatingSystem,
+  getVmStatus,
+  getVmiIpAddresses,
+  getWorkloadProfile,
+  getVmTemplate,
+  getTemplateDisplayName,
+  getNodeName,
+  getFlavor,
+  VmStatuses,
+  BootOrder,
+  isVmOff,
+} from 'kubevirt-web-ui-components';
+
+import { ResourceSummary, NodeLink, ResourceLink } from '@console/internal/components/utils';
+import { PodKind } from '@console/internal/module/k8s';
+import { getName, getNamespace } from '@console/shared';
+import { PodModel } from '@console/internal/models';
+
+import { VMKind, VMIKind } from './types';
+import { Dashes } from './utils';
+
+export const VMResourceSummary = ({ vm }: VMResourceSummaryProps) => {
+  const template = getVmTemplate(vm);
+  const templateLink = template && getTemplateDisplayName(template); // TODO(mlibra): link to a template detail, once implemented
+
+  return (
+    <ResourceSummary resource={vm}>
+      <dt>Description</dt>
+      <dd>{getDescription(vm)}</dd>
+      <dt>Operating System</dt>
+      <dd>{getOperatingSystemName(vm) || getOperatingSystem(vm) || <Dashes />}</dd>
+      <dt>Template</dt>
+      <dd>{templateLink || <Dashes />}</dd>
+    </ResourceSummary>
+  );
+};
+
+export const VMDetailsList = ({ vm, vmi, pods, migrations }: VMResourceListProps) => {
```

Comment:
are you planning to do the editing part as well? IMO it would be much easier to pull it in atomically than by parts.

Reply:
Editing will have to wait for a followup patch, we need to get at least this part in.

Reply:
ok